src/pages/profile/[user_id].tsx

```diff
@@ -162,7 +220,55 @@ export default function Home() {
               )}
             </Stack>
           </Card>
+
+          {/* Feedback */}
+          <Card
+            sx={{
+              width: "50vw",
+              minWidth: "300px",
+
+              marginBottom: "2vh",
+            }}
+          >
+            <CardContent style={{ paddingBottom: 0 }}>
+              {/* header */}
+              <Box display="flex" sx={{ alignItems: "center" }}>
+                <Box sx={{ flexGlow: 1 }}>
+                  <Typography variant="h2">Review</Typography>
+                </Box>
+                <ExpandMore
+                  expand={expanded}
+                  onClick={handleExpandClick}
+                  aria-expanded={expanded}
+                  aria-label="show more"
+                >
+                  <ArrowDownwardIcon color="secondary" />
+                </ExpandMore>
+              </Box>
+
+              <Collapse in={expanded} sx={{ marginTop: 2, marginBottom: 1 }}>
+                <FeedBackList feedbacks={feedbacks} />
+              </Collapse>
+            </CardContent>
+          </Card>
```

Comment:
add check verify condition to display this component since only hosts (verified users) have this feature on their profile.

Reply:
Added